src/Microsoft.ML.DnnAnalyzer/Microsoft.ML.DnnAnalyzer/DnnAnalyzer.cs

```diff
@@ -0,0 +1,37 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.ML.Runtime;
+using Microsoft.ML.Runtime.Data;
+using Microsoft.ML.Runtime.Internal.Utilities;
+using Microsoft.ML.Transforms.TensorFlow;
+using System.Linq;
+
+namespace Microsoft.ML.DnnAnalyzer
+{
+    public static class DnnAnalyzer
+    {
+        public static void Main(string[] args)
+        {
+            using (var env = new TlcEnvironment())
+            using (var ch = env.Start("DnnAnalyzer"))
+            {
+                if (Utils.Size(args) != 1)
+                {
+                    ch.Error("Usage: dotnet DnnAnalyzer.dll <model_location>");
+                    ch.Done();
+                    return;
+                }
+
+                foreach (var (name, opType, type, inputs) in TensorFlowUtils.GetModelNodes(args[0]))
```

Comment:
>foreach [](start = 16, length = 7)

I can add more arguments to this app, to let the user filter nodes by operation type (for example, sometimes there are lots of "Const" nodes that users might not be interested in if they are just trying to find the name of a certain layer).

Is this valuable? Is having a method that returns this information enough so users can filter programatically, or would user want this as well?